### ISSUE TITLE
Can we fine tune on unsupervised data, i.e bunch of PDF's 

### ISSUE THREAD

**Issue by SAIVENKATARAJU** [facebookresearch/DPR#170]

**Comment by vlad-karpukhin** [comment]
Yes you can if you can use some form of distant supervision - if you have answers, you can identify which passages to use as positives and which as negatives. Also, DPR is trained for passages of 100 words length, so it is recommended to split your PDFs into 100-word chunks

**Comment by SAIVENKATARAJU** [comment]
Hi
We have ~1k of documents and we plan to build a knowledge base model, where model is capable to pull the answer from the documents, consider this scenario, can we train this documents for better generating of embeddings.

**Comment by vlad-karpukhin** [comment]
1k might be quite small training dataset for domain adaptation.
If your domain differs from wikipedia, that won;t be enough and the dense IR will be worse than bm25 most likely.
You may look at some synthetic domain adaptation tricks here: https://dl.acm.org/doi/pdf/10.1145/3404835.3463085
